packages/atlas/src/components/_comments/CommentInput/CommentInput.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useRef, useState } from 'react'
+import mergeRefs from 'react-merge-refs'
+import useResizeObserver from 'use-resize-observer'
+
+import { Information } from '@/components/Information'
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+
+import { Border, ButtonsContainer, Container, Flex, StyledCommentRow, StyledTextArea } from './CommentInput.styles'
+
+import { CommentRowProps } from '../CommentRow'
+
+export type CommentInputProps = {
+  processing: boolean
+  onComment?: () => void
+  onCancel?: () => void
+} & CommentRowProps
+
+export const CommentInput: React.FC<CommentInputProps> = ({ processing, onCancel, onComment, ...rest }) => {
+  const smMatch = useMediaMatch('sm')
+  const containerRef = useRef<HTMLDivElement>(null)
+  const textAreaRef = useRef<HTMLTextAreaElement>(null)
+  const [active, setActive] = useState(false)
+  const [text, setText] = useState('')
+
+  const { ref: measureRef, height: textAreaHeight = 40 } = useResizeObserver({ box: 'border-box' })
```

Comment:
If I remember correctly how it works, you can pass ref to the `useResizesObserver` options. You don't need to create another `measureRef`.

Reply:
I tried that at first but it gave be buggy results where it would stop measuring this way is how I got it to work